### ISSUE TITLE
Warning: Failed prop type: The prop `id` is required to make `Dropdown` accessible for users of assistive technologies such as screen readers.

### ISSUE THREAD

**Issue by kasari** [kasari/react-admin-practice#2]
```
bundle.js:8938 Warning: Failed prop type: The prop `id` is required to make `Dropdown` accessible for users of assistive technologies such as screen readers.
    in Dropdown (created by Uncontrolled(Dropdown))
    in Uncontrolled(Dropdown) (created by NavDropdown)
    in NavDropdown (created by Header)
    in ul (created by Nav)
    in Nav (created by Header)
    in div (created by Grid)
    in Grid (created by Navbar)
    in nav (created by Navbar)
    in Navbar (created by Uncontrolled(Navbar))
    in Uncontrolled(Navbar) (created by Header)
    in Header (created by App)
    in div (created by App)
    in App (created by RouterContext)
    in RouterContext (created by Router)
    in Router
```

http://stackoverflow.com/questions/32510976/react-bootstrap-dropdown-with-input-wont-stay-open
